octobot_backtesting/data/database.py

```diff
@@ -139,10 +139,11 @@ async def select_from_timestamp(self, table, timestamps: list, operations: list,
                                                                         values=timestamps,
                                                                         operations=operations,
                                                                         should_quote_value=False)
-        timestamps_where_clauses = f"AND {timestamps_where_clauses}" if timestamps_where_clauses else ''
+        where_clause = self.__where_clauses_from_kwargs(**kwargs)
+        final_where_close = f"{where_clause} AND " if where_clause and timestamps_where_clauses else where_clause
+        final_where_close = f"{final_where_close}{timestamps_where_clauses}"
```

Comment:
Why did you split in two lines ?

Reply:
There was an algorithmic issue with the AND